tests/backup/backup_share_test.go

```diff
@@ -2358,6 +2359,9 @@ var _ = Describe("{ClusterBackupShareToggle}", func() {
 			_, err = CreateScheduleBackupWithValidation(ctx, scheduleName, backupClusterName, backupLocationName, backupLocationUID, scheduledAppContexts, labelSelectors, BackupOrgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of schedule backup with schedule name [%s]", scheduleName))
 			scheduleNames = append(scheduleNames, scheduleName)
+			log.InfoD("Getting the first schedule backup from admin")
```

Comment:
I think CreateScheduleBackupWithValidation returns firstScheduleBackupName.

Reply:
will take this in next PR since this is merged